src/modeci_mdf/mdf.py

```diff
@@ -68,7 +68,7 @@ class Function(MdfBase):
             https://mdf.readthedocs.io/en/latest/api/MDF_function_specifications.html
         args: Dictionary of values for each of the arguments for the Function, e.g. if the in-built function
               is linear(slope),the args here could be {"slope":3} or {"slope":"input_port_0 + 2"}
-        value: If the function is a value expression, this attribute will contain the expression and the function
+        valuess: If the function is a value expression, this attribute will contain the expression and the function
```

Comment:
just checking: is it `valuess` (two `s`?)

Reply:
i will edit it now